server/graphql/types/station.js

```diff
@@ -27,11 +29,24 @@ export const resolvers = {
     longitude ({ stop_lon }) {
       return stop_lon
     },
-    routes ({ stop_id }, {from, length}) {
-      return db().query(aql`
+    routes ({ stop_id }, { from, length }) {
+      return RoutesByStationIdLoader.load({ stop_id, from, length })
+    }
+  }
+}
+
+const getRoutesByStationIds = (stopIds) => {
+  const { from, length } = stopIds[0]
+  stopIds = map(stopIds, 'stop_id')
+
+  return new Promise((resolve, reject) => {
+    db().query(aql`
+      for stopId in ${stopIds}
+        let from = stopId.from
+
         let stops = (
           for stop in stops
-          filter stop.parent_station == ${stop_id}
+          filter stop.parent_station == stopId
```

Comment:
Je suis obliger de wraper avec un 'for' ce qu'attend dataloader en retour est une `Promise<array<value>>` et l'`array` doit être de la même taille que le tableau passer en paramètre de `const getRoutesByStationIds = (stopIds) => {`